gcovr/gcov.py

```diff
@@ -29,13 +29,17 @@
 
 output_re = re.compile(r"[Cc]reating [`'](.*)'$")
 source_re = re.compile(r"[Cc](annot|ould not) open (source|graph|output) file")
+unknown_cla_re = re.compile(r"Unknown command line argument")
 
 exclude_line_flag = "_EXCL_"
 exclude_line_pattern = re.compile(r'([GL]COVR?)_EXCL_(START|STOP)')
 
 c_style_comment_pattern = re.compile(r'/\*.*?\*/')
 cpp_style_comment_pattern = re.compile(r'//.*?$')
 
+gcov_cmd = None
+gcov_options = None
+
```

Comment:
I would prefer if this could be implemented without global variables.
But i am not sure where to put these cached variables.

This is a non-blocking comment, if there is no easy/better way to do tis.

Reply:
I think for this the command (this function) must be wrapped in a class with static members. I can take a deeper look after my holiday.